job_definitions/export_data.yml

```diff
@@ -1,5 +1,4 @@
 {% set environments = ['preview', 'production'] %}
-{% set disabled = 'true' %}
```

Comment:
We might have to explicitly set `disabled = 'false'` or remove the usage of it below.

Reply:
Good point, fixed now